examples/go-sdk-with-store/main.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2025 The OPA Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/open-policy-agent/opa/v1/bundle"
+	"github.com/open-policy-agent/opa/v1/sdk"
+	sdktest "github.com/open-policy-agent/opa/v1/sdk/test"
+
+	eopa_bundle "github.com/open-policy-agent/eopa/pkg/plugins/bundle"
+	eopa_sdk "github.com/open-policy-agent/eopa/pkg/sdk"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Use EOPA's bundle activator, ensuring EOPA's default
+	// storage layer will be used.
+	a := &eopa_bundle.CustomActivator{}
```

Comment:
Hmm we could add a helper to make this nicer, I would think. This line and the two following could be an exported function. What do you think?